api/spec-generator/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "aptos-api-spec-generator"
+version = "0.1.0"
+authors = ["Aptos Labs <opensource@aptoslabs.com>"]
+description = "Aptos API OpenAPI Spec Generator"
+repository = "https://github.com/aptos-labs/aptos-core"
+homepage = "https://aptoslabs.com"
+license = "Apache-2.0"
+publish = false
+edition = "2018"
```

Comment:
would recommend to put higher edition (I think we have 2021 or so somewhere in the root of the repo)

Reply:
Seems like 2018 is still the standard:
```
$ rg 'edition = ' | grep 2021 | wc -l
       2
$ rg 'edition = ' | grep 2018 | wc -l
     124
```